From 8dadac7cc8a2c50a12d429b4e5bb262d890aac36 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Tue, 6 Nov 2018 15:29:52 +0100
Subject: [PATCH 7/7] [SQUSH][DNM] Merge tag 'android-9.0.0_r16' into
 lineage-16.0

commit 1797c6976fdc4e475a5fcba18ede7fde395ea21b
Merge: 0673f0ba2 270e90f03
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Wed Aug 8 03:08:33 2018 +0000

    Snap for 4939496 from 270e90f033dd412206697e3a27aee76c24038c33 to pi-qpr1-release

    Change-Id: I58c840b601d31afc4b161d62a28052f71460e906

commit 270e90f033dd412206697e3a27aee76c24038c33
Author: Greg Kaiser <gkaiser@google.com>
Date:   Wed Aug 1 12:34:32 2018 -0700

    Revert "Support Speck encryption."

    This reverts commit 49c27c5cb2c48ce4665affe2b3086077c768a9ae.

    Remove the Speck encryption support. It was eventually
    decided not to allow Speck in Android P, so this code
    is no longer needed and wasn't used outside of testing.

    Bug: 112009351
    Test: Confirmed AES continues to work with FBE.
    Change-Id: Ia5458143be5687fff8d541d8fa2c8ee24a369da4

commit 0673f0ba26a7754494b013c95cdb8b342da23a87
Merge: 93d837f3a ae2e19dd9
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Fri Jul 27 03:14:25 2018 +0000

    Snap for 4916303 from ae2e19dd96e47a084eac09c3b3095b3bef427415 to pi-qpr1-release

    Change-Id: I35cf9d0f8977bc35e330ef1b91c207a27f4a4d82

commit ae2e19dd96e47a084eac09c3b3095b3bef427415
Author: Thierry Strudel <tstrudel@google.com>
Date:   Thu Jul 19 18:49:15 2018 -0700

    metricslogger: fix ACTION_BATTERY_CAUSED_SHUTDOWN not matching proto

    Bug: 111664962
    Test: "pixelstats_client -s" uses the right ATOM value
    Change-Id: I2944c6d9f79298ce88812fad218552b92afee97a
    Signed-off-by: Thierry Strudel <tstrudel@google.com>

commit 93d837f3a90acec007647f21ed4573f044fa6f1e
Author: Doheon Lee <doheon1.lee@lge.com>
Date:   Tue Jun 19 15:23:21 2018 +0900

    Determine product partition path on build time

    Path of product partitoin can be set as /product or /system/product
    whether generate extra product partition or not.
    Substitute %PRODUCT% to relevant path to know linker which path should
    search and permit.

    Bug: 110286945
    Test: m -j # Check /system/etc/ld.config.$(PLATFORM_VNDK_VERSION).txt
    Change-Id: I6ca177d0c9c5af00ad821879fece40848331fc8d
    Merged-In: I6ca177d0c9c5af00ad821879fece40848331fc8d
    (cherry picked from commit cccad0bf8461ab575a5861f9bc68c9f8f69db582)

commit c4cf62f4b4e9186a897e943b2419fb9a13739c1b
Author: Inseob Kim <inseob@google.com>
Date:   Mon Jun 18 13:21:45 2018 +0900

    Add /system/product/* paths to permitted paths

    As linker doesn't resolve paths in permitted paths, /system/product
    variants should be added to support devices having product partition
    under /system.

    Bug: 110286945
    Test: m -j succeeds on taimen and libraries under /system can dlopen
    libraries under /system/product/apps

    Change-Id: Icd102d44511702e4ec66c07a367b59c3d9700a44
    Merged-In: Icd102d44511702e4ec66c07a367b59c3d9700a44
    (cherry picked from commit 3918936b9e7fee51828d285a1217c7202d5cb2d1)

commit ee81388edb86973c81a26b0b97e957fa05425c64
Merge: 95d314be3 9a95028ea
Author: TreeHugger Robot <treehugger-gerrit@google.com>
Date:   Mon Jun 18 23:35:25 2018 +0000

    Merge "Revert "healthd: add stem to health@2.0-service.override [DO NOT MERGE]"" into pi-dev

commit 9a95028eac3bb01a4c04012a62a4588d9eed5558
Author: Yifan Hong <elsk@google.com>
Date:   Mon Jun 18 21:22:49 2018 +0000

    Revert "healthd: add stem to health@2.0-service.override [DO NOT MERGE]"

    This should have been done with actual fixes to sepolicy and rc scripts.

    This reverts commit 299276785787a35effa829f3f9d507cbde266dea.

    Reason for revert: broke the build

    Bug: 110228707

    Change-Id: I23944d044dd2f87e6c1033ed7a4c7cf34f16247e

commit 95d314be388b1cb28f4987937078a190d3da5c76
Merge: 74e8061fd 299276785
Author: Yifan Hong <elsk@google.com>
Date:   Mon Jun 18 20:15:24 2018 +0000

    Merge "healthd: add stem to health@2.0-service.override [DO NOT MERGE]" into pi-dev

commit 299276785787a35effa829f3f9d507cbde266dea
Author: Yifan Hong <elsk@google.com>
Date:   Thu Jun 14 15:12:26 2018 -0700

    healthd: add stem to health@2.0-service.override [DO NOT MERGE]

    The override module should have the same name as the -service
    module; the only difference is that healthd is installed or not.
    Without this change, the binary is completely useless due to
    incorrect rc files and sepolicy file_contexts.

    This allows emulator to use this module correctly.

    Bug: 110228707

    Test: builds on emulator, boots
    Change-Id: Id2062925ec3567c4fde70358184bb790541bf83b

commit 74e8061fd04c962d6e002109de57795eabd8f04b
Merge: f514f6f66 b8371d532
Author: TreeHugger Robot <treehugger-gerrit@google.com>
Date:   Thu Jun 14 17:29:54 2018 +0000

    Merge "Populate recovery DTBO offset correctly" into pi-dev

commit f514f6f66aba10f2668481a4d643c7983a0ea93a
Author: Hridya Valsaraju <hridya@google.com>
Date:   Wed May 30 10:57:18 2018 -0700

    Correct comment in boot image header

    Test: make
    Bug: 80207223
    Change-Id: I88cf7683e036ff43bbe3ff5418519c7e35e08b79
    Merged-In: I88cf7683e036ff43bbe3ff5418519c7e35e08b79
    (cherry picked from commit d0ab4e3701dcec866cb05f4d2a122fbbcabf61cc)

commit b8371d532d6cd1314954c2d3e9b8a79d3d70e7c7
Author: Hridya Valsaraju <hridya@google.com>
Date:   Thu May 31 12:39:58 2018 -0700

    Populate recovery DTBO offset correctly

    Also, remove recovery_dtbo_offset argument
    for mkbootimg as this is calculated based
    on sizes of kernel, ramdisk and second
    binaries.

    Also, modify unpack_bootimg to use the recovery_dtbo_offset
    field to extract recovery_dtbo.

    Test: make recoveryimage showcommands -j32
    Bug: 80207223
    unpack_bootimg --boot_img $OUT/recovery.img
    diff recovery_dtbo dtbo.img

    Change-Id: I588ccc8b739c169b6f78c17ffe554c5562397d98
    Merged-In: I588ccc8b739c169b6f78c17ffe554c5562397d98
    (cherry picked from commit 26e01bbdc5cb2dcad64d73ec836447b8ded4fc6c)

Change-Id: I170e00bba4dc20b82e7f5e469847dc7256133793
---
 libmetricslogger/include/metricslogger/metrics_logger.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libmetricslogger/include/metricslogger/metrics_logger.h b/libmetricslogger/include/metricslogger/metrics_logger.h
index c305db2da..88575be9d 100644
--- a/libmetricslogger/include/metricslogger/metrics_logger.h
+++ b/libmetricslogger/include/metricslogger/metrics_logger.h
@@ -114,7 +114,7 @@ enum {
     FIELD_BATTERY_RESISTANCE_UOHMS = 1448,
     FIELD_BATTERY_CURRENT_UA = 1449,
     FIELD_HARDWARE_LOCATION = 1450,
-    ACTION_BATTERY_CAUSED_SHUTDOWN = 1441,
+    ACTION_BATTERY_CAUSED_SHUTDOWN = 1451,
 };
 
 enum {
-- 
2.17.1

